.circleci/config.yml

```diff
@@ -252,9 +252,8 @@ workflows:
   # Build the Docker images and run Forge tests
   build-test-deploy:
     when:
-      or:
-        - equal: [auto, << pipeline.git.branch >>]
-        - equal: [canary, << pipeline.git.branch >>]
+      not:
+        equal: [main, << pipeline.git.branch >>]
```

Comment:
why remove canary?

Reply:
when not main, it runs on all other branches